jsettlers.logic/src/main/java/jsettlers/logic/map/grid/partition/PartitionsGrid.java

```diff
@@ -49,6 +42,14 @@
 import jsettlers.logic.player.PlayerSetting;
 import jsettlers.logic.player.Team;
 
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.*;
```

Comment:
Thx, just had to find out my IntelliJ forgot the auto formatter code style again...